docs/source/contributing.rst

```diff
@@ -399,3 +402,30 @@ and ``scico-data`` repositories must be updated and kept in sync.
    ::
 
       git submodule foreach --recursive 'git push' && git push
+
+
+Building Documentation
+----------------------
+
+A local copy of the documentation can be built from the respository root directory by doing
+
+::
+
+  python setup.py build_sphinx
+
+
+Alternatively:
+
+1. Navigate to the docs directory ``docs/``
+
```

Comment:
I've been changing this phrasing to "Change directory to ..." since "Navigate" seems to imply the use of a file manager, but we're referring here to the use of a terminal.

Reply:
Fixed it.

Reply:
Also changed other occurrences of "Navigate" to "Change directory" .